src/toolcalls/chain/chainsRegistry.ts

```diff
@@ -57,66 +56,82 @@ export const chainRegistry: ChainRegistry = {
       nativeToken: 'KAVA',
       nativeTokenDecimals: 18,
       blockExplorerUrls: ['https://kavascan.com/'],
+      extraTools: ['erc20-conversion'],
```

Comment:
You can define tools that _only_ get registered when certain chains are connected. These are defined in `extraTools`